bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -0,0 +1,146 @@
+      </div>
+    {% endif %}
+    </div>
+    <div id="version-message-desktop-old" class="version-message">
+      {{_('Looks like you’re using an older version of Firefox.')}}
+      <div class="version-message-secondary">
+        <a href="https://support.mozilla.org/kb/update-firefox-latest-version">{{_('Update to stay fast and safe.')}}</a>
+      </div>
+    </div>
+    <div id="version-message-linux-arm" class="version-message">
+      {{ _('Please follow <a href="%(url)s">these instructions</a> to install Firefox.')|format(url='https://support.mozilla.org/kb/install-firefox-linux') }}
+    </div>
+  </div>{#-- /.version-message-container --#}
+
+  <div id="main-feature" class="hide-for-refresh">
+    <div class="row">
+      <img src="{{ static('img/firefox/new/firefox-logo.png') }}" alt="Firefox" />
+      <h1 class="large" id="main-headline">
+        {{ _('Take Control.<br> Choose Firefox.') }}
+      </h1>
+    </div>
+  </div>{#-- /#main-feature --#}
+
+  <div class="desktop-latest-links-wrapper latest-links-wrapper">
+    <ul>
+      <li class="hide-for-refresh"><a href="{{ url('firefox.desktop.index') }}">{{_('Learn more about Firefox for desktop')}}</a></li>
+      <li><a href="https://support.mozilla.org/products/firefox?utm_source=mozilla.org&amp;utm_medium=referral&amp;utm_campaign=need-help-link">{{_('Need help?')}}</a></li>
+      {% if l10n_has_tag('mobile_links') %}
+      <li class="hide-for-refresh"><a href="{{settings.GOOGLE_PLAY_FIREFOX_LINK }}">{{_('Get Firefox on your Android device')}}</a></li>
+      {% if l10n_has_tag('firefox_ios') %}
+      <li class="hide-for-refresh"><a href="{{ firefox_ios_url('mozorg-uptodate-appstore-link') }}">{{ _('Add Firefox to your iPhone, iPad and iPod touch') }}</a></li>
+      {% endif %}
+      <li class="hide-for-refresh"><a href="{{ url('firefox.os.index') }}">{{_('Learn about Firefox OS')}}</a></li>
+      {% endif %}
+      <li><a href="{{ firefox_url('desktop', 'all') }}#{{ LANG }}">{{_('Download a fresh copy')}}</a></li>
+    </ul>
+  </div>
+  <div class="android-latest-links-wrapper latest-links-wrapper">
+    <ul>
+      <li><a href="{{ url('firefox.android.index') }}">{{_('Learn more about Firefox for Mobile')}}</a></li>
+    </ul>
+  </div>
+  <div class="fxos-latest-links-wrapper latest-links-wrapper">
+    <ul>
+      <li><a href="{{ url('firefox.os.index') }}">{{_('Learn about Firefox OS')}}</a></li>
+      <li><a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}">{{_('Discover all the ways to take Firefox on the go')}}</a></li>
+    </ul>
+  </div>
+
+  <section id="download-buttons">
+    {# handles desktop users (windows, os x, linux) #}
+    <div class="download-button-wrapper desktop" id="download-button-wrapper-desktop">
+      {{ download_firefox(simple=true, dom_id='primary-download-button') }}
```

Comment:
Note: changing the Id of the primary download button will cause one of our sanity tests to fail here: https://github.com/mozilla/bedrock/blob/master/tests/functional/firefox/test_new.py#L17

Unless there is a reason to change it, suggest just using the default ID that's already in place? This would then mean we don't have to skip that particular test.